packages/superset-ui-connection/src/types.ts

```diff
@@ -11,11 +11,30 @@ export type FetchRetryOptions = {
 };
 export type Headers = { [k: string]: string };
 export type Host = string;
+
+export type JsonPrimitive = string | number | boolean | null;
+/**
```

Comment:
You may want to put this before the `StrictJsonObject` line. 
Currently, it will only appear in auto-complete of `StrictJsonValue`.